src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/ApplicationModelConventionExtensions.cs

```diff
@@ -50,6 +50,60 @@ public static class ApplicationModelConventionExtensions
             conventions.Add(new ActionApplicationModelConvention(actionModelConvention));
         }
 
+        /// <summary>
+        /// Adds a <see cref="IParameterModelConvention"/> to all the parameters in the application.
+        /// </summary>
+        /// <param name="conventions">The list of <see cref="IApplicationModelConvention"/>
+        /// in <see cref="AspNetCore.Mvc.MvcOptions"/>.</param>
+        /// <param name="parameterModelConvention">The <see cref="IParameterModelConvention"/> which needs to be
+        /// added.</param>
+        public static void Add(
+            this IList<IApplicationModelConvention> conventions,
+            IParameterModelConvention parameterModelConvention)
+        {
+            conventions.Add(new ParameterApplicationModelConvention(parameterModelConvention));
```

Comment:
Should I also add null checks for the `Add` method with `IActionModelConvention` in this PR?